JSDemos/Demos/TreeList/LoadDataOnDemand/description.md

```diff
@@ -1 +1,14 @@
-This example demonstrates how to&nbsp;load data on&nbsp;demand every time a&nbsp;user expands a&nbsp;TreeList node. For this purpose, filtering is&nbsp;delegated to&nbsp;the server, and IDs of&nbsp;expanded nodes are sent to&nbsp;the server with every request for data.
\ No newline at end of file
+The TreeList can load a remote dataset dynamically as a user expands nodes. The dataset must have a [plain structure](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#dataStructure).
+
+This feature requires client- and server-side configurations. To configure the client-side part, do the following:
+
+1. **Send an expanded node's ID to the server**       
+For this, implement the CustomStore's [load](/Documentation/ApiReference/Data_Layer/CustomStore/Configuration/#load) function. In this demo, we do it in the [dataSource](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#dataSource) configuration object.
+
+1. **Delegate filtering to the server**         
+Set the **remoteOperations**.[filtering](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/remoteOperations/#filtering) property to **true**.
+
+3. **Specify the data field that defines whether the node has children**           
+Use the [hasItemsExpr](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#hasItemsExpr) property to set this data field.
+
+Server-side implementation is available in the [ASP.NET Core](/Demos/WidgetsGallery/Demo/DataGrid/RemoteReordering/NetCore/Light/) and [ASP.NET MVC](/Demos/WidgetsGallery/Demo/DataGrid/RemoteReordering/Mvc/Light/) versions of this demo under the `TreeListDataController.cs` tab.
```

Comment:
This mode will not work if a customer completes the mentioned steps because they also need to implement the back-end logic. If they use ASP.NET on the server, they can use the provided code from the `TreeListDataController.cs` tab to implement the server-side logic.

Reply:
So "Server-side implementation for ASP.NET is available in the [ASP.NET Core]() and [ASP.NET MVC]() versions of this demo under the `TreeListDataController.cs` tab."?

Reply:
Alright, thanks, I get it. In this case, you may want to convey this idea a bit clearer. For example:

>Now you need a server-side implementation. For instance, you can use ASP.NET Core and ASP.NET MVC versions of this demo under the `TreeListDataController.cs` tab.